Headers/DebugServer2/Target/ProcessBase.h

```diff
@@ -13,14 +13,14 @@
 
 #include "DebugServer2/Target/ProcessDecl.h"
 #include "DebugServer2/Target/ThreadBase.h"
+#include "DebugServer2/SoftwareBreakpointManager.h"
 
 #include <functional>
 #include <set>
 
 namespace ds2 {
 
-class BreakpointManager;
-class WatchpointManager;
+class HardwareBreakpointManager;
```

Comment:
Well it's needed for the API. I could do more in this commit, but it was really intended as just a "rename the api to be more accurate" commit.